tsl/test/expected/dist_api_calls.out

```diff
@@ -205,3 +205,11 @@ _timescaledb_internal._dist_hyper_1_8_chunk
  
 (1 row)
 
+-- Ensure that move_chunk() and reorder_chunk() functions cannot be used
+-- with distributed hypertable
+\set ON_ERROR_STOP 0
+SELECT move_chunk(chunk=>'_timescaledb_internal._dist_hyper_1_4_chunk', destination_tablespace=>'tablespace1', index_destination_tablespace=>'tablespace1', reorder_index=>'disttable_time_idx', verbose=>TRUE);
+ERROR:  tablespace "tablespace1" does not exist
```

Comment:
Can you move to an existing tablespace to verify the behavior is blocked.

Reply:
Oh yes, missed that one